content/docs/user-guide/packaging/project-export/project-export-ios.md

```diff
@@ -0,0 +1,98 @@
+---
+linkTitle: Project Export iOS
+title: Project Export for iOS
+description: (Experimental) Learn how to use the Project Export CLI to create an Xcode project to build and deploy for iOS.
+toc: true
+weight: 450
+---
+This guide covers how to use project export tooling to build and deploy games on iOS. First, we will use the iOS export script to generate an Xcode project and prepare assets. Afterwards we will walk through how to ensure Xcode is properly setup to build an iOS app.
+
+For this demonstration, we will use the O3DE Atom Sample Viewer project as our reference point for ensuring proper behavior. Results may vary for other projects.
+
+{{< note >}}
+To learn more about the project export tooling, please consult the page [Project Export CLI Tool.](project-export-cli)
+{{< /note >}}
+{{< important >}}
+The iOS export functionality is only available on macOS, which is currently experimental for O3DE.
+{{< /important >}}
+
+## Prerequisites
+1. Complete the [Project Export CLI Tool page.](project-export-cli)  prerequisites. 
+2. Git clone [O3DE Atom Sample Viewer project](https://github.com/o3de/o3de-atom-sampleviewer) onto your machine.
+3. Register [O3DE Atom Sample Viewer project](https://github.com/o3de/o3de-atom-sampleviewer) with O3DE. You can register the project using:
+```
+<O3DE_ENGINE>/scripts/o3de.sh register -pp /path/to/o3de-atom-sampleviewer
```

Comment:
its not always obvious what `<O3DE_ENGINE>` means. I don't see any explanation in this document. This needs to be pointed out, and maybe use an environment variable prefix `$` (for mac) instead. 
```
    export O3DE_ENGINE=$HOME/o3de
```
or something.